apps/src/netsim/NetSimShard.js

```diff
@@ -38,16 +38,25 @@ var NetSimShard = module.exports = function (shardID, pubSubConfig) {
   var channel = this.pubSub.subscribe(shardID);
 
   /** @type {NetSimTable} */
-  this.nodeTable = new NetSimTable(channel, shardID, 'n');
+  this.nodeTable = new NetSimTable(shardID, 'n', {
+    channel: channel
```

Comment:
I agree; having a required parameter in "options" feels wrong. I see that we have jshint's `maxparams` set to three, and I assume that's the reason for the change, but I would argue that's a bit low.